### ISSUE TITLE
Problem: Decision points break with 10 choices or more

### ISSUE THREAD

**Issue by replaceafill** [archivematica/Issues#850]
**Expected behaviour**

The `Actions` dropdown of a decision point works with any number of options.

**Note** I modified the options of this particular job in `workflow.json` to make the problem visible.

![transfer-js-modified](https://user-images.githubusercontent.com/560781/62578177-c1899780-b85d-11e9-9949-a8cb2592e9e1.png)

**Current behaviour**

When the `Actions` dropdown has 10 choices or more, either in the transfer tab or in the ingest tab, and you mouse over it to pick a choice a gray overlay is displayed which doesn't allow you to do anything else.

![broken-layout](https://user-images.githubusercontent.com/560781/62578592-bbe08180-b85e-11e9-8efb-49e49bda6a98.png)

This is supposed to be a modal form that allows you to search/autocomplete the many options. Tweaking the CSS of the page a bit you can see some of it:

![broken-overlay-displayed](https://user-images.githubusercontent.com/560781/62578920-9c962400-b85f-11e9-9d35-1f283727fe26.png)

[This forum post](https://groups.google.com/forum/#!topic/archivematica/5EHxf9nueTc) has a video showing the issue in a customized workflow.

**Steps to reproduce**

I modified the options of the `f09847c2-ee51-429a-9478-a860477f6b8d` job (`Do you want to perform file format identification?`) in the `MCPServer/lib/assets/workflow.json` file to have more than 10 options.

**Your environment (version of Archivematica, OS version, etc)**

https://github.com/artefactual-labs/am/commit/9fb9106aedea9d9cc25a8f90c336ee2beea8d268

---
**For Artefactual use:**
Please make sure these steps are taken before moving this issue from Review to Done:

- All PRs related to this issue are properly linked 👍
- All PRs related to this issue have been merged 👍
- Test plan for this issue has been implemented and passed 👍
- Documentation regarding this issue has been written and it has been added to the release notes, if needed 👍

**Comment by FransPicturae** [comment]
I have fixed this behaviour in a pull-request. The dropdown has a class 'hide' that prevents it to display. Some styling is added also.

**Comment by sromkey** [comment]
Adding the triage label for the next release since we have a PR to review :)

**Comment by ross-spencer** [comment]
Adding a sample portion of a processing config to copy and paste for the chain-link referenced: 
```json
        "f09847c2-ee51-429a-9478-a860477f6b8d": {
            "config": {
                "@manager": "linkTaskManagerReplacementDicFromChoice",
                "@model": "MicroServiceChoiceReplacementDic",
                "replacements": [
                    {
                        "description": {
                            "en": "Yes",
                            "es": "Sí",
                            "fr": "Oui",
                            "ja": "はい",
                            "pt_BR": "Sim",
                            "sv": "Ja"
                        },
                        "id": "d97297c7-2b49-4cfe-8c9f-0613d63ed763",
                        "items": {
                            "IDCommand": "True"
                        }
                    },
                    {
                        "description": {
                            "en": "No 1",
                            "fr": "Non",
                            "ja": "いいえ",
                            "sv": "Nej"
                        },
                        "id": "1f77af0a-2f7a-468f-af8c-653a9e61ca4f",
                        "items": {
                            "IDCommand": "False"
                        }
                    },
                    {
                        "description": {
                            "en": "No 2",
                            "fr": "Non",
                            "ja": "いいえ",
                            "sv": "Nej"
                        },
                        "id": "1f77af0a-2f7a-468f-af8c-653a9e61ca4f",
                        "items": {
                            "IDCommand": "False"
                        }
                    },
                    {
                        "description": {
                            "en": "No 3",
                            "fr": "Non",
                            "ja": "いいえ",
                            "sv": "Nej"
                        },
                        "id": "1f77af0a-2f7a-468f-af8c-653a9e61ca4f",
                        "items": {
                            "IDCommand": "False"
                        }
                    },
                    {
                        "description": {
                            "en": "No 4",
                            "fr": "Non",
                            "ja": "いいえ",
                            "sv": "Nej"
                        },
                        "id": "1f77af0a-2f7a-468f-af8c-653a9e61ca4f",
                        "items": {
                            "IDCommand": "False"
                        }
                    },
                    {
                        "description": {
                            "en": "No 5",
                            "fr": "Non",
                            "ja": "いいえ",
                            "sv": "Nej"
                        },
                        "id": "1f77af0a-2f7a-468f-af8c-653a9e61ca4f",
                        "items": {
                            "IDCommand": "False"
                        }
                    },
                    {
                        "description": {
                            "en": "No 6",
                            "fr": "Non",
                            "ja": "いいえ",
                            "sv": "Nej"
                        },
                        "id": "1f77af0a-2f7a-468f-af8c-653a9e61ca4f",
                        "items": {
                            "IDCommand": "False"
                        }
                    },
                    {
                        "description": {
                            "en": "No 7",
                            "fr": "Non",
                            "ja": "いいえ",
                            "sv": "Nej"
                        },
                        "id": "1f77af0a-2f7a-468f-af8c-653a9e61ca4f",
                        "items": {
                            "IDCommand": "False"
                        }
                    },
                    {
                        "description": {
                            "en": "No 8",
                            "fr": "Non",
                            "ja": "いいえ",
                            "sv": "Nej"
                        },
                        "id": "1f77af0a-2f7a-468f-af8c-653a9e61ca4f",
                        "items": {
                            "IDCommand": "False"
                        }
                    },
                    {
                        "description": {
                            "en": "No 9",
                            "fr": "Non",
                            "ja": "いいえ",
                            "sv": "Nej"
                        },
                        "id": "1f77af0a-2f7a-468f-af8c-653a9e61ca4f",
                        "items": {
                            "IDCommand": "False"
                        }
                    },
                    {
                        "description": {
                            "en": "No 10",
                            "fr": "Non",
                            "ja": "いいえ",
                            "sv": "Nej"
                        },
                        "id": "1f77af0a-2f7a-468f-af8c-653a9e61ca4f",
                        "items": {
                            "IDCommand": "False"
                        }
                    },
                    {
                        "description": {
                            "en": "No 11",
                            "fr": "Non",
                            "ja": "いいえ",
                            "sv": "Nej"
                        },
                        "id": "1f77af0a-2f7a-468f-af8c-653a9e61ca4f",
                        "items": {
                            "IDCommand": "False"
                        }
                    },
                    {
                        "description": {
                            "en": "No 12",
                            "fr": "Non",
                            "ja": "いいえ",
                            "sv": "Nej"
                        },
                        "id": "1f77af0a-2f7a-468f-af8c-653a9e61ca4f",
                        "items": {
                            "IDCommand": "False"
                        }
                    },
                    {
                        "description": {
                            "en": "No 13",
                            "fr": "Non",
                            "ja": "いいえ",
                            "sv": "Nej"
                        },
                        "id": "1f77af0a-2f7a-468f-af8c-653a9e61ca4f",
                        "items": {
                            "IDCommand": "False"
                        }
                    },
                    {
                        "description": {
                            "en": "No 14",
                            "fr": "Non",
                            "ja": "いいえ",
                            "sv": "Nej"
                        },
                        "id": "1f77af0a-2f7a-468f-af8c-653a9e61ca4f",
                        "items": {
                            "IDCommand": "False"
                        }
                    },
                    {
                        "description": {
                            "en": "No 15",
                            "fr": "Non",
                            "ja": "いいえ",
                            "sv": "Nej"
                        },
                        "id": "1f77af0a-2f7a-468f-af8c-653a9e61ca4f",
                        "items": {
                            "IDCommand": "False"
                        }
                    }
                ]
            },
            "description": {
                "en": "Do you want to perform file format identification?"
            },
            "exit_codes": {
                "0": {
                    "job_status": "Completed successfully",
                    "link_id": "c3269a0a-91db-44e8-96d0-9c748cf80177"
                }
            },
            "fallback_job_status": "Failed",
            "fallback_link_id": "61c316a6-0a50-4f65-8767-1f44b1eeb6dd",
            "group": {
                "en": "Identify file format",
                "es": "Identificar formato de fichero",
                "fr": "Identifier le format de fichier",
                "pt_BR": "Identifique o formato do arquivo",
                "sv": "Identifiera filformat"
            }
        },
```

**Comment by sallain** [comment]
I tested this on qa/1.x on my docker environment.

1. I modified `workflow.json` to add 15 extra dropdown options to the `Perform file format identification (Transfer)` job, as per @ross-spencer's comment above. (Ignore the floating dropdown box, it's just my screencap app acting up.)

![image](https://user-images.githubusercontent.com/4612276/75480911-fc1c6400-5956-11ea-9777-b3db2662a4b3.png)

2. I started a transfer and had it stop at the file identification microservice on the Transfer tab for manual entry.

3. I moused over the dropdown box. A popup appeared on the screen containing a responsive dropdown.

![image](https://user-images.githubusercontent.com/4612276/75481539-130f8600-5958-11ea-9c11-82dd0bf31422.png)

4. I selected my choice and the transfer continued.

A few times, when I selected my choice, it didn't register with Archivematica - it seemed to happen if I took more than a few seconds to make a choice, so maybe related to the page refresh. If I select the choice pretty quickly (within a few seconds) it triggers properly. I don't think this is a blocker.

I did the same test with the `Perform file format identification (Ingest)` job and experienced the same results.

Overall, this seems to be working as expected.
